README.md

```diff
@@ -9,6 +9,26 @@ Major features include:
 - Interactive console with auto-completion
 - Requires no special permissions
 
+## Getting Started
+Install the plugin using the Notepad++ Plugin Manager, or download it from the [Release](https://github.com/dail8859/LuaScript/releases) page and copy `LuaScript.dll` to your `\plugins` folder.
+
+#### Startup Scripts
+LuaScript looks for the file `\plugins\config\startup.lua` and automatically runs it on Notepad++ startup. You can include any commands you want to immediately execute on prgram startup, as well as register any additional functions/shortcuts.
```

Comment:
Currently the most "Lua"-ish way is using `require`. I understand this won't pick up on new scripts like some other methods would. 

> include some code in the base plugin which does this cleanly for a specified folder.

I had considered this but I'm still hoping to either support external Lua modules or possibly ship this with LFS which would provide a much cleaner solution.